Procurement/ViewModel/Filters/ForumExport/LegionFilter.cs

```diff
@@ -0,0 +1,16 @@
+﻿using POEApi.Model;
```

Comment:
If this file has only `IncubatorFilter`, it should be named `IncubatorFilter.cs`.

Reply:
Done